interface/src/Menu.cpp

```diff
@@ -772,6 +772,7 @@ Menu::Menu() {
     // Help > About High Fidelity
     action = addActionToQMenuAndActionHash(helpMenu, "About High Fidelity");
     connect(action, &QAction::triggered, [] {
+        DependencyManager::get<OffscreenUi>()->clearCache();
```

Comment:
I assume this was for testing? please remove it since it could result in problems.

Reply:
All done.